### PR TITLE
email: send email to librarian on user deletion with ongoing loans

### PR DESCRIPTION
**Pull request by KonstantinaStoikou**
Add function that sends emails to management email of library with ongoing loans of a patron.

references https://github.com/CERNDocumentServer/cds-books/issues/53

### CODE REVIEW

File: invenio_app_ils/acquisition/jsonresolvers/order_order_lines.py [hunk 1]
```diff
@@ -22,36 +23,38 @@ def jsonresolver_loader(url_map):
     """Resolve the referred document and patron for an Order Line."""
     from flask import current_app
 
-    def patron_resolver(order_line, patron):
+    def patron_resolver(order_line, patron_pid):
         """Resolve the Patron for the given Order Line."""
-        order_line["patron"] = patron.dumps_loader()
+        patron = get_patron_or_unknown(patron_pid)
+
+        order_line["patron"] = patron
```
Comment (zzacharo) [inline invenio_app_ils/acquisition/jsonresolvers/order_order_lines.py]: I know it's one line and convenient but this implicit modification here can lead to nasty errors...I would say maybe better rename that function to `order_line_patron_resolver` or `set_order_line_patron`. WDYT?

File: invenio_app_ils/circulation/search.py [hunk 2]
```diff
@@ -138,3 +138,10 @@ def get_loans_by_patron_pid(patron_pid):
         "term", patron_pid=patron_pid
     )
     return search
+
+
+def get_active_loans_by_patron_pid(patron_pid):
+    """Returns all the ongoing loans for a given patron."""
```
Comment (zzacharo) [inline invenio_app_ils/circulation/search.py]: minor: Maybe a small refactoring in the docstring as it doesn't return the results i.e the ongoing loans: 
```suggestion
    """Constructs the search object for all the ongoing loans of a given patron."""
```
@kprzerwa WDYT?

File: invenio_app_ils/ext.py [hunk 3]
```diff
@@ -209,7 +209,7 @@ def init_app(self, app):
         )
         app.register_blueprint(
             Blueprint(
-                "invenio_app_ils_circulation_ mail",
+                "invenio_app_ils_circulation_mail",
```
Comment (zzacharo) [inline invenio_app_ils/ext.py:212]: Wait this is outdated no? I remember adding this through an entrypoint in the end, no?

File: invenio_app_ils/acquisition/jsonresolvers/order_order_lines.py (hunk 1 above)
Comment (kpsherva) [reply invenio_app_ils/acquisition/jsonresolvers/order_order_lines.py]: @zzacharo what kind of errors do you mean? just for us to better remember this example
in general lets discuss this together

File: invenio_app_ils/ext.py (hunk 3 above)
Comment (KonstantinaStoikou) [reply invenio_app_ils/ext.py:212]: The entrypoint didn't work.

**Review by kpsherva — APPROVED** [review]

**Review by zzacharo — APPROVED** [review]

**Pull request merged**